content/soarian-financials/stu3/individuals/patient.md

```diff
@@ -0,0 +1,638 @@
+
+The following fields are returned if valued:
+
+   * [Patient name](http://hl7.org/fhir/patient-definitions.html#Patient.name){:target="_blank"}
+   * [Patient id](http://hl7.org/fhir/patient-definitions.html#Patient.identifier){:target="_blank"}
+   * [Extensions including birth time, preferred telecom, bad address indicator](http://hl7.org/fhir/extensibility-registry.html)
+   * [Contact information (includes phone,faxand email)](http://hl7.org/fhir/datatypes.html#ContactPoint){:target="_blank"}
+   * [Gender (administrative)](http://hl7.org/fhir/patient-definitions.html#Patient.gender){:target="_blank"}
+   * [Date of Birth](http://hl7.org/fhir/patient-definitions.html#Patient.birthDate){:target="_blank"}
+   * [Deceased](http://hl7.org/fhir/patient-definitions.html#Patient.deceased_x_){:target="_blank"}
+   * [Address](http://hl7.org/fhir/patient-definitions.html#Patient.contact.address){:target="_blank"}
+   * [Communication (preferred language)](http://hl7.org/fhir/patient-definitions.html#Patient.communication){:target="_blank"}
+   * [Marital status](http://hl7.org/fhir/patient-definitions.html#Patient.maritalStatus){:target="_blank"}
+   * [Primary Care Providers](http://hl7.org/fhir/patient-definitions.html#Patient.generalPractitioner){:target="_blank"}
+ 
+
+## Extensions
+* [Time of day of birth]
+* [Patient level user data](#patient-user-data)
+* [Bad address indicator](http://hl7.org/fhir/v3/AddressUse/cs.html){:target="_blank"}
+* [Preferred telecom](http://hl7.org/fhir/extension-iso21090-preferred.html){:target="_blank"}
+
+## Search
+
+Search for Patients that meet supplied query parameters:
+
+    GET /Patient/<Soarian Finanicals Person Number>
+
+_Implementation Notes_
+
+* The [Patient.animal] modifier element is not supported and will not be returned.
+* Direct secure email will not be returned.
+* Only Active patients will be returned.
+* Patient Specific Persosn (PSP) will not be returned.
+* If the patient being searched is a person then the information is retured as per the person FHIR format https://www.hl7.org/fhir/person.html
+* Information that are stored as collections such as name, address,identifiers etc. will be retured only of there is an active entry aviabale.
+* Only active official (legal) name, address and identifiers will be retured.
+
+
+
+
+### Example
+
+#### Request
+
+	GET https://sfet-nprodnstg.dm50.dev.smed.net/sf-qa1451/dynamic/services/regsched/stu3/v1/patient/104136
+
+#### Response
+
+##### Header
+	Status: 200 OK
+	Location: https://DEV3SRF0191.dm50.dev.smed.net/sf-qa1451/dynamic/services/regsched/stu3/v1/Patient/104136
+		
+##### Body		
```

Comment:
use the `<%= json(:dstu2_patient_bundle) %>` macro

you will need to add a file `soarian_financials_examples_patient.rb` to lib/resources/example_json

invoke the macro like `<%= json(:soarian_financials_patient_entry) %>` (assuming you need to use entry because this is for a read, bundle would be the right name for a search)

Reply:
Would we want to create a new macro for stu3 or is it ok to continue using dstu2_patient_bundle?

Reply:
the macro is the json part, and you can pass in a symbol named whatever you add to the example_json file, so yeah, name that something like soarian_financials_patient_entry (use entry for a read, bundle for a search)